course-content/33_create_reports.qmd

```diff
@@ -0,0 +1,431 @@
+
+We see that here with the format option. This option is where we tell Quarto what types of formats we wish to produce, and for each format, how we wish to produce it. In this case, we have our four formats, each with different sub-options. For html, we indicated that we wish for `toc` to be true. In this context, that means we wish to include a table of contents with the final output - this is populated with our markdown headers and sections. Next, for the pdf format, we specified that upon execution of a code cell, we should *not* echo the code into the report (i.e: hide all the code, but display the outputs of the code). After this, we included the option `reference-doc` for pptx. In this case, it means that we have a template file from wish we wish to base our powerpoint on - it is located in the same folder with the name `template.potx`. You may either add your own template file to the folder, or change the pptx option to match that of the final format docx. In this last case, we have simply specified to use all of Quarto's default options for a word document to see what it will produce. 
+
+All of these are fairly simplified examples of formatting options that Quarto provides. For more detailed explanations, examples, and tutorials, check out the [incredible guides Quarto has to offer!](https://quarto.org/docs/guide/) 
+
+### Simple Example
+
+Let's now populate our notebook with some Markdown and code cells. You may copy and paste each of these blocks of code into your own notebook, changing your cells to markdown where necessary.
+
+```md
+## First Section
+
+We can write some documentation for our document in the **markdown** portion of a notebook.
+```
+
+```python
+# next include some code!
+
+import numpy as np
+import matplotlib.pyplot as plt
+```
+
+```python
+# we can break up the code over multiple cells
+
+x = np.random.normal(size=1000) # list of 1000 random values between (0,1)
+
+# create a plot
+plt.hist(x)
+plt.show()
+```
+
+```md
+## Second Section
+
+### With Subsection
+
+Each of these sections will show up in the table of contents.
+```
+
+This next section of code has special comments at the beginning of the code, starting with `#|`. This indicates that we are passing additional quarto arguments to this cell only (unlike the global format options we chose above). In this case, we are creating a plot and giving it a caption. 
+
+:::{.callout-tip}
+## All Code Block Execution Options
+
+Here's a list of all the standard execution options we can pass to a codeblock using `#|`:
+
+| Option | Description |
+| ------ | ----------- |
+| `eval` | Evaluate the code chunk (if false, just echos the code into the output). |
+| `echo` | Include the source code in output. |
+| `output` | Include the results of executing the code in the output (true, false, or asis to indicate that the output is raw markdown and should not have any of Quarto’s standard enclosing markdown). |
```

Comment:
```suggestion
| `output` | Include the results of executing the code in the output (true, false, or as is to indicate that the output is raw markdown and should not have any of Quarto’s standard enclosing markdown). |
```

Reply:
Is actually one word - will change to code backticks to be clear.